src/index.ts

```diff
@@ -61,7 +61,10 @@ export default class NewRelicLambdaLayerPlugin {
       return;
     }
 
-    const plugins = _.get(this.serverless, "service.plugins", []);
+    let plugins = _.get(this.serverless, "service.plugins", []);
+    if (typeof plugins === "object") {
+      plugins = plugins.modules;
+    }
```

Comment:
One thing that comes to mind is that typeof [] is "object," so this might not work as intended.  Perhaps a check on whether or not the plugins variable has length?

Reply:
Or perhaps
```
    if (typeof plugins === "object" && plugins.modules) {
      plugins = plugins.modules;
    }
```